src/views/FindOrder.vue

```diff
@@ -104,75 +104,94 @@
           <!-- Order Item Section -->
           <hr />
 
-          <div @click="() => router.push('/order')">
+          <div v-for="(order, index) in orders" :key="index" :order="order" @click="() => router.push(`/order/${order.doclist.docs[0].orderId}`)">
             <section class="section-header">
               <div class="primary-info">
                 <ion-item lines="none">
                   <ion-label>
-                    Order ID
-                    <p>Customer Name</p>
+                    {{ order.doclist.docs[0].orderId }}
```

Comment:
I think orderId should be available directly on order object and we should perform such extraction in actions. Same applies to other fields here

Reply:
Extracted the commonly used fields in action and then used them directly.